Formula/dynare.rb

```diff
@@ -0,0 +1,72 @@
+class Dynare < Formula
+  desc "Platform for economic models, particularly DSGE and OLG models"
+  homepage "https://www.dynare.org/"
+  url "https://www.dynare.org/release/source/dynare-4.5.3.tar.xz"
+  sha256 "01434f6d3ceaff1891dc771f6a5b39caee787b2ffa1875e5a4e8c673e32ff3d7"
+
+  head do
+    url "https://github.com/DynareTeam/dynare.git"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "bison" => :build
+    depends_on "flex" => :build
+  end
+
+  depends_on "boost" => :build
+  depends_on "cweb" => :build
+  depends_on "fftw"
+  depends_on "gcc" # for gfortran
+  depends_on "gsl"
+  depends_on "hdf5"
+  depends_on "libmatio"
+  depends_on "metis"
+  depends_on "octave"
+  depends_on "suite-sparse"
+  depends_on "veclibfort"
+
+  needs :cxx11
+
+  resource "slicot" do
+    url "https://mirrors.ocf.berkeley.edu/debian/pool/main/s/slicot/slicot_5.0+20101122.orig.tar.gz"
+    mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/s/slicot/slicot_5.0+20101122.orig.tar.gz"
+    sha256 "fa80f7c75dab6bfaca93c3b374c774fd87876f34fba969af9133eeaea5f39a3d"
+  end
+
+  def install
+    ENV.cxx11
+
+    resource("slicot").stage do
+      system "make", "lib", "OPTS=-fPIC", "SLICOTLIB=../libslicot_pic.a",
+             "FORTRAN=gfortran", "LOADER=gfortran"
+      system "make", "clean"
+      system "make", "lib", "OPTS=-fPIC -fdefault-integer-8",
+             "FORTRAN=gfortran", "LOADER=gfortran",
+             "SLICOTLIB=../libslicot64_pic.a"
+      (buildpath/"slicot").install "libslicot_pic.a", "libslicot64_pic.a"
+    end
+
+    system "autoreconf", "-fvi" if build.head?
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--disable-matlab",
+                          "--with-slicot=#{buildpath}/slicot"
+    system "make", "install"
+  end
+
+  def caveats; <<~EOS
+    To get started with dynare, open Octave and type:
+
+            addpath #{opt_lib}/dynare/matlab
```

Comment:
@ilovezfs Plz fix this massive indentation before merging 🙈.

Reply:
don't worry. nothing is getting merged until `--HEAD` passes the test block

Reply:
Maybe we should add an `opt_pkglib` for `lib/name` 😈.